### DISCUSSION TITLE
Hide and Show Menu by Setting Status to True False

### DISCUSSION THREAD

**Discussion by QuangLanh** [clientIO/joint#2453] [Q&A]
### Introduction

How to hide this left menu by setting true false state?

### Steps to reproduce

I want to put a variable in the code that displays the menu on the left. If it is true, the menu will appear, and if it is false, it will be hidden so that the interface containing the paper can be larger.

![hidden sidebar](https://github.com/clientIO/joint/assets/91791283/6606fd3b-3457-4d72-bf90-3e6219680528)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Here's an example of a collapsible stencil: https://www.jointjs.com/demos/chatbot
It's ok to destroy (on collapse) and re-create (on expand) the stencil.
